src/server/ProcessList.cpp

```diff
@@ -30,51 +27,27 @@ using namespace Microsoft::Console::Interactivity;
 [[nodiscard]] HRESULT ConsoleProcessList::AllocProcessData(const DWORD dwProcessId,
                                                            const DWORD dwThreadId,
                                                            const ULONG ulProcessGroupId,
-                                                           _In_opt_ ConsoleProcessHandle* const pParentProcessData,

```

Comment:
`pParentProcessData` wasn't used for anything except to change the return value.